pandas/core/generic.py

```diff
@@ -7607,9 +7607,10 @@ def between_time(
         self: FrameOrSeries,
         start_time,
         end_time,
-        include_start: bool_t = True,
-        include_end: bool_t = True,
+        include_start=lib.no_default,
+        include_end=lib.no_default,
```

Comment:
I think there's a way to annotate these, can you check other functions that use `lib.no_default`?

Reply:
I'm not sure what you're asking me to look for?
Is it something along the lines of this? `include_start: bool | NoDefault = no_default,`

Reply:
yes, that looks right